Create a suggested dessert that doesn't contain eggs.

elsabagh opened the pull request:
a suggested dessert that doesn't contain eggs

a mechanism has been implemented to prevent duplication of the same suggestion.

elsabagh committed:
Create a suggested dessert that doesn't contain eggs.

A mechanism has been implemented to prevent duplicate suggestions.

Al-Taie commented on src/main/kotlin/presentation/FoodChangeModeUI.kt:
```diff
@@ -49,26 +47,21 @@ class FoodChangeMoodUI(
+    private fun printEggFreeSweet() {
+        try {
+            val meal = getEggFreeSweetUseCase()
+            println("\nEgg-Free Sweet:")
+            println("Name: ${meal.name}")
+            println("Minutes: ${meal.minutes}")
+            println("Submitted: ${meal.submitted}")
+            println("Nutrition: ${meal.nutrition}")
+            println("StepsCount: ${meal.stepsCount}")
+            println("Steps: ${meal.steps}")
+            println("Description: ${meal.description.take(150)}...")
+            println("Ingredients: ${meal.ingredients}")
+            println("IngredientsCount: ${meal.ingredientsCount}")
```
Use formatted print like the old project (print the info as a table)

elsabagh committed:
Create a suggested meal containing more than 700 calories and The comment was resolved in the suggested dessert task that does not contain eggs.

- A mechanism has been implemented to prevent duplicate suggestions.

- Like/dislike mechanism used
